plugin/postgres/plugin.go

```diff
@@ -22,6 +22,7 @@
 //        "pg_password":"password-for-above-user",
 //        "pg_host":"hostname-or-ip-of-pg-server",
 //        "pg_port":"port-above-pg-server-listens-on"
```

Comment:
Can you add a comma to the end of this comment, so it generates valid json, now that you've added a new field?